app/src/main/java/com/example/bigowlapp/repository/exception/PhoneNumberTakenException.java

```diff
@@ -0,0 +1,14 @@
+package com.example.bigowlapp.repository.exception;
+
+public class PhoneNumberTakenException extends Exception {
```

Comment:
This is great for debugging and for letting us send an appropriate error message in the UI.